test/Sema/enum_equatable_hashable.swift

```diff
@@ -13,8 +13,8 @@ enum Generic<T> {
   case A, B
 
   func method() -> Int {
-    if A == B { }
-    return A.hashValue
+    if .A == .B { }
```

Comment:
This isn't really testing the same thing; it can pick *any* `==`. Can you make the method static instead?